app/api/components/route.ts

```diff
@@ -0,0 +1,14 @@
+import { NextResponse } from "next/server";
+import { checkFlag } from "@formBuilder/actions";
+
+//Seperated from utils to avoid unavailable dependencies in testing and response downloads.
+export const allowAddressComplete = async () => {
+  return checkFlag("addressComplete");
+};
+
+// This route is used to allow Cypress Testing of components, and provide the API key for AddressComplete.
```

Comment:
I wonder if there is anything we should do to secure this endpoint? It just gets a flag and key so not from a privacy/* concern. But I wonder if it's worth mentioning to Pat - do we need to add any new rules/*